bin/sdl.cc

```diff
@@ -230,9 +231,6 @@ int main(int argc, char** argv)
 
     //_______________________________________________________________________________
     // --lower_level
-#ifdef CUT_VALUE_DEBUG
```

Comment:
@YonsiG Is this OK?

Reply:
Hi Philip, I suggest we keep this flag without removing it. Because once we use the CUT_VALUE_DEBUG, we create more branches than the usual output, we save the cut values and a few chi2s which is not needed for the usual output but still under tuning.

Reply:
@sgnoohc, Yanxi  asked that we keep this flag but I believe this has not been reverted. Is it in the plans?

I saw a lot of function deletions in the writeNTuple. I hope they are not deleting functions related to this?

In any case, before merging, I would ask @YonsiG and @bsathian, who are working on the tuning of more low level stuff to run this branch to check that everything they need for their studies is still included.